### PR TITLE
Add target attribute to megamenu for external links

### PR DESCRIPTION
**Pull request by RobAWilkinson**

### CODE REVIEW

File: src/components/navigation/MegaMenu/Column.jsx [hunk 1]
```diff
@@ -61,6 +65,7 @@ Column.propTypes = {
     link: PropTypes.shape({
       text: PropTypes.string.isRequired,
       href: PropTypes.string.isRequired,
+      target: PropTypes.string,
```
Comment (ncksllvn) [inline src/components/navigation/MegaMenu/Column.jsx]: Do we need to have a default value applied to the `target` property?

Comment (RobAWilkinson) [reply src/components/navigation/MegaMenu/Column.jsx]: good call, I'll set it as `_self` in my tests if it was undefined, react just wasnt applying the attribute however

Comment (ncksllvn) [reply src/components/navigation/MegaMenu/Column.jsx]: Ah nice, totally slipped my mind that that's how React would handle it.

**Comment by 1Copenut** [comment]
Did we want to add a `rel="noopener noreferrer" declaration also? Thinking of this article when I see `_target`: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

**Comment by RobAWilkinson** [comment]
I'd never seen this before @1Copenut  we have a ton of `target="_blanks"` without it scattered throughout the site.  

Since the pages that we are linking too are trusted VA content that we control, and not user-generated seems like it won't affect us.

**Pull request merged**